src/visualization/types/Graph/view.tsx

```diff
@@ -276,6 +298,12 @@ const XYPlot: FC<Props> = ({properties, result, timeRange, annotations}) => {
       }
     })
 
+    if (inAnnotationWriteMode) {
+      config.interactionHandlers = {
+        singleClick: makeSingleClickHandler(),
+      }
+    }
```

Comment:
it doesn't seem to affect things, but why this change in position?

Reply:
Just helped me keep track of this as i was reading